test/Dss-C/TestDssC.c

```diff
@@ -7,6 +7,57 @@
 
 
 int runTheTests();
+int ReadGrids(const char* file1);
+
+int TodaysTest() {
```

Comment:
I'd rename this test to be something more descriptive.

Reply:
I removed TodaysTest()